src/morello-install/README.md

```diff
@@ -593,7 +643,7 @@ option from the menu and press Space.
 Otherwise, press Enter to complete the installation:
 ```
 ┌───────────────────────────Final Configuration──────────────────────────────┐
-│ Setup of your FreeBSD system is nearly complete. You can now modify your   │
+│ Setup of your FreeBSD system is nearly complete. You can now modify your   │─
```

Comment:
Was this actually present or was it a typo?

Reply:
I think actually present -- I suspect that’s the overwritten screen header saying that it’s the FreeBSD installer, which gets a bit occluded in later output. To avoid confusion, I’ve deleted it, however.